crates/op-hardforks/src/optimism/sepolia.rs

```diff
@@ -0,0 +1,12 @@
+/// Canyon sepolia hardfork activation timestamp
+pub(crate) const OP_SEPOLIA_CANYON_TIMESTAMP: u64 = 1_699_981_200;
```

Comment:
```suggestion
pub const OP_SEPOLIA_CANYON_TIMESTAMP: u64 = 1_699_981_200;
```
let's make these pub